spec/requests/mutations/resend_confirmation_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'Resend confirmation' do
+  include_context 'with graphql query request'
+
+  let(:user)     { create(:user, confirmed_at: nil) }
+  let(:email)    { user.email }
+  let(:redirect) { Faker::Internet.url }
+  let(:query) do
+    <<-GRAPHQL
+      mutation {
+        userResendConfirmation(
+          email:"#{email}",
+          redirectUrl:"#{redirect}"
+        ) {
+          message
+          success
+        }
+      }
+    GRAPHQL
+  end
+
+  context 'when params are correct' do
+    it 'sends an email to the user with confirm url' do
+      expect { post_request }.to(
+        change(ActionMailer::Base.deliveries, :count).by(1)
+      )
+
+      email = Nokogiri::HTML(ActionMailer::Base.deliveries.last.body.encoded)
+      link  = email.css('a').first
+
+      # TODO: Move to feature spec
+      expect do
+        get link['href']
+        user.reload
+      end.to change(user, :confirmed_at).from(NilClass).to(Time)
```

Comment:
I grabbed this piece of code from another spec and adapted it for my own use. I saw you have a `TODO` for this. Not sure what your plan is here but make note that I'm using something similar here.